plugins/google/compute/InstanceTemplateMachineTypes.spec.js

```diff
@@ -0,0 +1,247 @@
+            const callback = (err, results) => {
+                expect(results.length).to.be.above(0);
+                expect(results[0].status).to.equal(0);
+                expect(results[0].message).to.equal('Google Cloud Virtual Machine instance template has the desired machine type');
+                done()
+            };
+
+            const cache = createCache(
+                [
+                    {
+                        "id": "864700679362969633",
+                        "creationTimestamp": "2021-06-25T04:32:30.481-07:00",
+                        "name": "instance-template-1",
+                        "description": "",
+                        "properties": {
+                            "machineType": "e2-micro",
+                            "canIpForward": false,
+                            "networkInterfaces": [
+                                {
+                                    "network": "https://www.googleapis.com/compute/v1/projects/akhtar-dev-aqua/global/networks/default",
+                                    "name": "nic0",
+                                    "accessConfigs": [
+                                        {
+                                            "type": "ONE_TO_ONE_NAT",
+                                            "name": "External NAT",
+                                            "networkTier": "PREMIUM",
+                                            "kind": "compute#accessConfig"
+                                        }
+                                    ],
+                                    "kind": "compute#networkInterface"
+                                }
+                            ],
+                            "disks": [
+                                {
+                                    "type": "PERSISTENT",
+                                    "mode": "READ_WRITE",
+                                    "deviceName": "instance-template-1",
+                                    "index": 0,
+                                    "boot": true,
+                                    "initializeParams": {
+                                        "sourceImage": "projects/debian-cloud/global/images/debian-10-buster-v20210609",
+                                        "diskSizeGb": "10",
+                                        "diskType": "pd-balanced"
+                                    },
+                                    "autoDelete": true,
+                                    "kind": "compute#attachedDisk"
+                                }
+                            ],
+                            "metadata": {
+                                "fingerprint": "8czNZABQPuc=",
+                                "kind": "compute#metadata"
+                            },
+                            "serviceAccounts": [
+                                {
+                                    "email": "779980017373-compute@developer.gserviceaccount.com",
```

Comment:
We never want to make account details public
```suggestion
                                    "email": "111122223333@developer.gserviceaccount.com",
```

Reply:
You can also remove unnecessary data as well. Maybe just keep the data which is needed for plugin